amqp/src/test/java/docs/javadsl/AmqpDocsTest.java

```diff
@@ -26,6 +27,8 @@
 import akka.stream.testkit.javadsl.TestSink;
 import akka.testkit.javadsl.TestKit;
 import akka.util.ByteString;
+import scala.collection.JavaConverters;
```

Comment:
This looks wrong here (and unused AFAICS).

Reply:
Right, removed.